week-2/G-conditionals-2/exercise-3.js

```diff
@@ -9,7 +9,10 @@
 */
 
 function calculateGrade(mark) {
-
+  if (mark > 80) return "A";
+  else if (mark >= 60 && mark <= 80) return "B";
```

Comment:
This is correct! 🎉 

But there are a few improvements we could make here. Usually when we use an `if` statement we also want to use curly braces (`{` or `}`).

The other thing is that we can make our `else if` statement a little shorter. You're checking that `mark` is greater than 80 first, so that means that if it reaches the `else if` then `mark` cannot be greater than 80.

That means that checking that `mark` is less than 80 in the `else if` is unnecessary.